.github/automation/aarch64/wiki_utils.py

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env python
+
+# *******************************************************************************
+# Copyright 2025 Arm Limited and affiliates.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# *******************************************************************************
+import argparse
+import ctest_utils
+import os
+
+PAGE_TITLE = "AArch64 Status"
```

Comment:
(nit) Generated page has heading duplicating the page name:
<img width="585" height="210" alt="image" src="https://github.com/user-attachments/assets/5f85bfde-b84e-471c-9aba-0bd1af5d9288" />

I would also suggest renaming the page to "AArch64 test results" or something similar for clarity.